src/features/offer/components/OfferBody/OfferBody.deprecated.native.test.tsx

```diff
@@ -115,7 +115,7 @@ describe('<OfferBody />', () => {
     expect(api.getnativev1offersreports).toBeCalled()
   })
 
-  it('should not request /native/v1/offers/reports if user is logged in and not connected', async () => {
+  xit('should not request /native/v1/offers/reports if user is logged in and not connected', async () => {
```

Comment:
it?

Reply:
Alors ce test est pété et j'ai du mal à comprendre pourquoi ce serait à `OfferBody` d'en subir les frais ^^
En effet on teste des conditions du hook `useReportedOffers()`. Ce hook a une logique en interne, et c'est `OfferBody.tsx` qui se charge de tester cette logique interne.

Du coup, puisque ce n'est pas vraiment lié à `OfferBody.tsx` et que le test est pété je l'ai désactivé.
Je pense qu'il faudrait avoir un `useReportedOffers.test.ts` et le tester donc unitairement.

Reply:
`xit` c'est l'équivalent d'un `it.skip` ?

Reply:
Oui